.github/workflows/ci.yaml

```diff
@@ -0,0 +1,83 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  industrial_ci:
+    strategy:
+      matrix:
+        env:
+          - ROS_DISTRO: galactic
```

Comment:
Should we also test for foxy? I see foxy is included in the pre-release yaml.

Reply:
removed foxy from pre-release testing... this repo is only targeting galactic and rolling as it uses c++17